Source/Orts.Simulation/Simulation/RollingStocks/MSTSWagon.cs

```diff
@@ -472,14 +472,9 @@ public virtual void LoadFromWagFile(string wagFilePath)
 
             var couplerlength = ((CarCouplerFaceLengthM - CarBodyLengthM) / 2) + 0.1f; // coupler length at rest, allow 0.1m also for slack
 
-            if (CarAirHoseLengthM < couplerlength)
+            if (CarAirHoseHorizontalLengthM == 0)
             {
-                CarCouplerFaceLengthM = CarBodyLengthM + (0.4f * 2.0f); // Assume a coupler length of 400mm at each end and add to car body length
-
-                if (Simulator.Settings.VerboseConfigurationMessages)
-                {
-                    Trace.TraceInformation("Coupler length exceeded brake air hose length, so  ORTSLengthCouplerFace decreased to {0}", CarCouplerFaceLengthM);
-                }
+                CarAirHoseHorizontalLengthM = 0.3862f; // 15.2 inches
```

Comment:
Could you extend the comment to link to the source for the 15.2" dimension? - E.g. a forum thread?

Reply:
I don't have a solid reference for this. 

It was done to suit the airhoses used in the demonstration.